src/govuk/components/input/input.yaml

```diff
@@ -13,7 +13,7 @@ params:
   description: Type of input control to render. Defaults to "text".
 - name: inputmode
   type: string
-  require: false
+  required: false
```

Comment:
I love it when you improve your test coverage and it catches stuff like this! 👏🏻 

Great job.